### PR TITLE
[13.0][FIX][queue_job] Update Multi-Company Access Rules To 13.0

### PR DESCRIPTION
**Pull request by Tatider**

### CODE REVIEW

**Comment by Tatider** [comment]
Odoo SA also removed patch for `JSONEncoder.default` for [Odoo 12.0](https://github.com/odoo/odoo/commit/ac78a9fab9109f63bcdc149556e1010cdcb6e417#diff-b61f597e5df5c60179429d7364d8a170). If changes to fix it are fine, then they should be passed to 12.0 as well.

File: queue_job/migrations/13.0.1.2.0/noupdate_changes.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8" ?>
```
Comment (astirpe) [inline queue_job/migrations/13.0.1.2.0/noupdate_changes.xml]: This file `noupdate_changes.xml` seems not necessary anymore and could be removed.

**Review by astirpe — APPROVED** [review]
LGTM!

File: queue_job/migrations/13.0.1.2.0/noupdate_changes.xml (hunk 1 above)
Comment (Tatider) [reply queue_job/migrations/13.0.1.2.0/noupdate_changes.xml]: Removed

**Review by simahawk — APPROVED** [review]
thanks!

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**